sdk/appcenter-identity/src/main/java/com/microsoft/appcenter/identity/Identity.java

```diff
@@ -380,6 +394,34 @@ public void run() {
         });
     }
 
+    private synchronized void instanceSignOut() {
+        if (mTokenStorage.getToken() == null) {
+            AppCenterLog.warn( LOG_TAG, "Couldn't sign-out: authToken doesn't exist.");
```

Comment:
```suggestion
            AppCenterLog.warn(LOG_TAG, "Couldn't sign-out: authToken doesn't exist.");
```

Reply:
also should be `sign out` since it's a verb - `sign-out` is when it's used as a noun